ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-master.yaml

```diff
@@ -31,6 +31,10 @@ tests:
   commands: make test-e2e
   openshift_installer_src:
     cluster_profile: aws
+- as: e2e-aws-image-ecosystem
+  commands: TEST_SUITE=openshift/image-ecosystem run-tests
+  openshift_installer:
+    cluster_profile: aws
```

Comment:
@gabemontero since the samples operator is responsible for installing the imagestreams+templates that imageeco tests, it makes sense to me that this job should be run against PRs for the samples operator.